dataactbroker/fileRoutes.py

```diff
@@ -130,6 +130,70 @@ def get_protected_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.get_protected_files()
 
+    @app.route("/v1/check_current_page/", methods=["GET"])
+    @convert_to_submission_id
```

Comment:
I'm not sure about this but currently, we are using `submission_id` which requires login. Do we want to allow no permission accesses?